frontend/src/components/print/config/ActivityConfig.vue

```diff
@@ -8,7 +8,7 @@
 
 <script>
 export default {
-  name: 'ActivityConfig',
+  name: 'Activity',
```

Comment:
Shouldn't this name match the filename? And according to the very first essential rule in the [Vue Style Guide](https://v2.vuejs.org/v2/style-guide/?redirect=true#Priority-A-Rules-Essential-Error-Prevention), component names should consist of multiple words.
As far as I see you only changed this to have shorter translation keys, but I would be fine with the longer ones as well.

Reply:
Generally, I agree with your comment regarding the style guide. That's the reason why I previously changed this from `Activity` to `ActivityConfig` in this commit: https://github.com/ecamp/ecamp3/commit/272382ee8275d979697b2060b88c4ea0f7d6833b

However, this broke the print config, because the component name is used as type property in the generated config json. So either we change the config interface to use `type: ActivityConfig`, or we need a separate property/function to return the type.

Reply:
Well, then maybe we are mis-using the component name in this case. You could create a base component or mixin or composable which is included in all print config components, and which just provides a method `configName` or `jsonName` or something similar. That method could map `ActivityConfig` to `Activity` etc. via a regex or substring operation, and it could still be overridden in the actual config components, if we ever need it.

Reply:
Fixed with https://github.com/ecamp/ecamp3/pull/2562/commits/61343a10e8b066b9afbe50cc922e8c61d34d3385